js/_footer/footer.js

```diff
@@ -1,7 +1,10 @@
 const footerTemplate = document.createElement('template');
-footerTemplate.innerHTML = `<footer class="feather mb-8 mt-4 flex flex-justify-center flex-items-center px-8 text-center">
+footerTemplate.innerHTML = `<footer class="feather mb-8 mt-16 flex flex-justify-center flex-items-center px-8 text-center">
```

Comment:
hey, instead of innerHTML, you can use Dom APIs. https://betterprogramming.pub/dom-manipulation-the-dangers-of-innerhtml-602f4119d905. hope this helps.

Reply:
Thanks for the suggestion. As long as it is just a simple line of HTML, it should not cause any issue, even the article you mentioned states that.